### PR TITLE
Remove fontawesome files

### PR DESCRIPTION
**Pull request by ilja-git**

### CODE REVIEW

File: frontend/init-pro-icons.sh [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+VERSION="6.4.0"
+
+unpack() {
+  name=${1}
+  npm pack "@fortawesome/${name}@${VERSION}"
+  mkdir -p "node_modules/@fortawesome/${name}"
+  tar xz --strip-components 1 -C "node_modules/@fortawesome/${name}" -f "fortawesome-${name}-${VERSION}.tgz"
+  rm "fortawesome-${name}-${VERSION}.tgz"
```
Comment (akheron) [inline frontend/init-pro-icons.sh:18]: `npm pack` + `tar x` combo seems unnecessary, perhaps just `npm install --no-save ${name}@${VERSION}` would be enough?

File: frontend/unpack-pro-icons.sh [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-
-unpack() {
-  NAME=${1}
-  mkdir -p node_modules/@fortawesome/"${NAME}"
-  tar xz --strip-components 1 -C node_modules/@fortawesome/"${NAME}" -f vendor/fortawesome/fortawesome-"${NAME}"-"${VERSION}".tgz
-}
-
-unpack pro-light-svg-icons
-unpack pro-regular-svg-icons
-unpack pro-solid-svg-icons
```
Comment (akheron) [inline frontend/unpack-pro-icons.sh:17]: This script needs to be kept for backwards compatibility, until other cities change their way of installing the pro icons.

File: frontend/init-pro-icons.sh (hunk 1 above)
Comment (ilja-git) [reply frontend/init-pro-icons.sh]: Tested `npm install`. The `package.json` contains `link:` -dependencies that are not compatible with node 18. Also one common package is installed with `npm install`. Using pack + tar for now.

File: frontend/unpack-pro-icons.sh (hunk 2 above)
Comment (ilja-git) [reply frontend/unpack-pro-icons.sh]: renamed to setup-pro-icons.sh and brought back init/unpack scripts.

**Review by akheron — APPROVED** [review]

**Pull request merged**